app/controllers/spina/admin/registers_controller.rb

```diff
@@ -53,6 +55,15 @@ def set_breadcrumb
         add_breadcrumb t('spina.registers.scaffold_name_plural'), spina.admin_registers_path
       end
 
+      def initialize_client
```

Comment:
Could we move this out into a shared helper method as it is identical in both usages?